inc/compatibility/woocommerce.php

```diff
@@ -225,6 +227,29 @@ function () {
 		}
 	}
 
+	/**
+	 * Normally, WooCommerce doesn't show the regular price on the cart, just shows the sale price.
+	 * With that, the regular price is shown also as crossed on the cart.
+	 *
+	 * @param  string $price that current price column value on the cart page.
+	 * @param  array  $cart_item current cart item.
+	 * @param  string $cart_item_key cart item key.
+	 * @return string
+	 */
+	public function show_crossed_regular_price_on_cart( $price, $cart_item, $cart_item_key ) {
+		$product = $cart_item['data'];
+
```

Comment:
@mskapusuz I think we can just use `return $product->get_price_html();` and that will be enough.